packages/react/src/Details/Details.tsx

```diff
@@ -40,22 +39,12 @@ const Root = React.forwardRef<HTMLDetailsElement, DetailsProps>(
       }
     }, [])
 
-    if (sxProp !== defaultSxProp) {
-      return (
-        <Box as={'details'} className={clsx(className, classes.Details)} {...rest} sx={sxProp} ref={ref}>
-          {/* Include default summary if summary is not provided */}
-          {!hasSummary && <Details.Summary data-default-summary>{'See Details'}</Details.Summary>}
-          {children}
-        </Box>
-      )
-    }
-
     return (
-      <details className={clsx(className, classes.Details)} {...rest} ref={ref}>
+      <BoxWithFallback as={'details'} className={clsx(className, classes.Details)} {...rest} ref={ref}>
```

Comment:
```suggestion
      <BoxWithFallback as="details" className={clsx(className, classes.Details)} {...rest} ref={ref}>
```

Super minor, not very important lol